mods/ts/rules/gdi-structures.yaml

```diff
@@ -433,14 +433,13 @@ GAPLUG:
 		Footprint: xxx xxx
 		Dimensions: 2,3
 	WithIdleOverlay@DISH:
-		RequiresCondition: !build-incomplete
```

Comment:
This looks like a rebase error, we don't want the overlays to show and animate before the buildup is complete.
Please revert this.